keras_hub/src/utils/preset_utils_test.py

```diff
@@ -59,6 +59,46 @@ def test_sharded_weights(self):
         ):
             self.assertAllClose(v1, v2)
 
+    @pytest.mark.large
+    def test_sharded_weights_config(self):
+        if not sharded_weights_available():
+            self.skipTest("Sharded weights are not available.")
+
+        init_kwargs = {
+            "vocabulary_size": 1024,
+            "num_layers": 12,
+            "num_query_heads": 8,
+            "num_key_value_heads": 4,
+            "hidden_dim": 32,
+            "intermediate_dim": 64,
+            "head_dim": 4,
+            "sliding_window_size": 5,
+            "attention_logit_soft_cap": 50,
+            "final_logit_soft_cap": 30,
+            "layer_norm_epsilon": 1e-6,
+            "query_head_dim_normalize": False,
+            "use_post_ffw_norm": True,
+            "use_post_attention_norm": True,
+            "use_sliding_window_attention": True,
+        }
+        backbone = GemmaBackbone(**init_kwargs)  # ~422KB
```

Comment:
Can you move this to `setUp` and use it for all 3 test cases which we are doing here, so that our test setup will be lot more cleaner.